riscv/src/lib.rs

```diff
@@ -40,6 +40,7 @@ pub(crate) mod bits;
 pub mod delay;
 pub mod interrupt;
 pub mod register;
+pub use riscv_pac::result;
```

Comment:
It could be handy to reexport the `riscv-pac` crate
```suggestion
pub use riscv_pac::{self as pac, result};
```

Reply:
... or even `pub use riscv_pac::*`?